ports/geckolib/string_cache/lib.rs

```diff
@@ -117,8 +117,11 @@ impl WeakAtom {
 
     #[inline]
     pub fn eq_str_ignore_ascii_case(&self, s: &str) -> bool {
+        let bytes = s.as_bytes();
         unsafe {
-            Gecko_AtomEqualsUTF8IgnoreCase(self.as_ptr(), s.as_ptr() as *const _, s.len() as u32)
+            Gecko_AtomEqualsUTF8IgnoreCase(self.as_ptr(),
+                                           bytes.as_ptr() as *const _,
+                                           bytes.len() as u32)
```

Comment:
That's true, I removed it. I'll also fill follow-ups to remove the FFI
functions for string comparison and a few other things.

On 08/13/2016 09:47 AM, Simon Sapin wrote:
> In ports/geckolib/string_cache/lib.rs
> <https://github.com/servo/servo/pull/12815#discussion_r74689195>:
> 
>>          unsafe {
>> -            Gecko_AtomEqualsUTF8IgnoreCase(self.as_ptr(), s.as_ptr() as *const _, s.len() as u32)
>> +            Gecko_AtomEqualsUTF8IgnoreCase(self.as_ptr(),
>> +                                           bytes.as_ptr() as *const _,
>> +                                           bytes.len() as u32)
> 
> I think this change is a no-op: |str::len| returns a number of UTF-8
> bytes, not of code points.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/servo/servo/pull/12815/files/93e6fb1a6569a2b6934cb28c8d4889122d0b2897..22a7892934b829e32c75a8e408b5569535f13414#r74689195>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABQwurP0WCuwUHsCBBfOmvp2EkqOgZ76ks5qffUtgaJpZM4Jh0-a>.
>